addons/customGear/uniforms/CfgVehicles.hpp

```diff
@@ -0,0 +1,190 @@
+        uniformClass = "potato_u_g3cw_serp_m05wdl_v2";
+        hiddenSelections[] = {
+            "Camo1",
+            "Camo2",
+            "Camo3",
+            "Camo4",
+            "noglove",
+            "grade",
+            "sang",
+            "drapeau",
+            "insignia",
+            "clan"
+        };
+        hiddenSelectionsTextures[] = {
+            QPATHTOEF(customGear,uniforms\data\ptv_jacket_m05_v2_co.paa),
+            QPATHTOEF(customGear,uniforms\data\G3_PANT_M05_v2_CO.paa),
+            "\ptv_characters\uniforms\Data\m77_co.paa",
+            "\ptv_characters\uniforms\data\usp_gloves_co.paa",
+            "",
+            ""
+        };
+        hiddenSelectionsMaterials[] = {"\ptv_characters\uniforms\data\jacket_summer.rvmat"};
+    };
+    class potato_s_g3cw_serp_m05wdl_v3: ptv_s_m08 {
+        author = "AMF/VityaB/Siege-A & PTV/Chesheire";
+        scope = 2;
+        model = "\ptv_characters\uniforms\ptv_Uniform_07.p3d";
+        modelSides[] = {3,1};
+        nakedUniform = "U_BasicBody";
+        uniformClass = "potato_u_g3cw_serp_m05wdl_v3";
+        hiddenSelections[] = {
+            "Camo1",
+            "Camo2",
+            "Camo3",
+            "Camo4",
+            "noglove",
+            "grade",
+            "sang",
+            "drapeau",
+            "insignia",
+            "clan"
+        };
+        hiddenSelectionsTextures[] = {
+            QPATHTOEF(customGear,uniforms\data\ptv_jacket_m05_v3_co.paa),
+            QPATHTOEF(customGear,uniforms\data\G3_PANT_M05_v2_CO.paa),
+            "\ptv_characters\uniforms\Data\m77_co.paa",
+            "\ptv_characters\uniforms\data\usp_gloves_co.paa",
+            "",
+            ""
+        };
+        hiddenSelectionsMaterials[] = {"\ptv_characters\uniforms\data\jacket_summer.rvmat"};
+    };
+    class potato_s_g3cw_serp_m05wdlwint: ptv_s_m08 {
+        author = "AMF/VityaB/Siege-A & PTV/Chesheire";
+        scope = 2;
+        model = "\ptv_characters\uniforms\ptv_Uniform_07.p3d";
+        modelSides[] = {3,1};
+        nakedUniform = "U_BasicBody";
+        uniformClass = "potato_u_g3cw_serp_m05wdlwint";
+        hiddenSelections[] = {
+            "Camo1",
+            "Camo2",
+            "Camo3",
+            "Camo4",
+            "noglove",
+            "grade",
+            "sang",
+            "drapeau",
+            "insignia",
+            "clan"
+        };
+        hiddenSelectionsTextures[] = {
+            QPATHTOEF(customGear,uniforms\data\ptv_jacket_m05_v2_co.paa),
+            QPATHTOEF(customGear,uniforms\data\G3_PANT_M05Wint_CO.paa),
+            "\ptv_characters\uniforms\Data\m77_co.paa",
+            "\ptv_characters\uniforms\data\usp_gloves_co.paa",
+            "",
+            ""
+        };
+        hiddenSelectionsMaterials[] = {"\ptv_characters\uniforms\data\jacket_summer.rvmat"};
+    };
+    class potato_s_g3cw_serp_m05wdlwint_v2: ptv_s_m08 {
+        author = "AMF/VityaB/Siege-A & PTV/Chesheire";
+        scope = 2;
```

Comment:
I think these should be scope1 but not totally sure
usually the weapon is scope2 and the man/vic thing is just scope1

Reply:
Pretty sure you're right - fixed!